mis_builder_analytic_filter/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+MIS Builder Analytic Filter
+===========================
+
+This module extends the functionality of mis_builder to
+provide filtering on analytic axis.
+
+Installation
+============
+
+There is no specific installation procedure for this module.
+
+Configuration and Usage
+=======================
+
+In Accounting > Reporting > MIS Reports, there is a new field
+'Analytic Account' which can be used to filter move line on an
+analytic axis and it's children.
+
+This filter is also available on the MIS Report widget, so the user
+can change the filter in the preview and dashboard views.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/91/8.0
+
+Known issues / Roadmap
+======================
+
+While useful as is, we are aware that such analytic filtering may need
+to be customized heavily depending on the customer context. This module can
+be extended or considered as an example.
+
+* at the moment, no filtering is done on other queries than move lines
```

Comment:
s/at/At
Add period
or
Currently, only filtering on move line queries is performed.